src/Particles/babylon.particleSystem.ts

```diff
@@ -109,7 +109,48 @@
          */
         public getClassName(): string {
             return "ParticleSystem";
-        }        
+        }
+
+        private _imageProcessingConfigurationDefines = new ImageProcessingConfigurationDefines();

```

Comment:
Why not having it on the base class?

Reply:
Because I did not notice there was a base class :-) I am changing it now.